### DISCUSSION TITLE
error : monogame not found

### DISCUSSION THREAD

**Discussion by kanatos-kun** [MonoGame/MonoGame#8283] [General]
Hi i've got a problem while following the tutorial for installing monogame
I've: 
- installed visual studio community 2022 
- installed monogame with the extension
- got this error when doing a new project
![Capture d’écran 2024-04-17 20 42 00](https://github.com/MonoGame/MonoGame/assets/22226839/3ceb5082-67c6-401e-8583-91ed6d28f720)

what i've done :
- desinstall the extension and reinstall
- desinstall visual studio community 2022 and reinstall
- desinstall monogame et reinstall via command prompt

i'm on window 10 64bit

**Comment by BlizzCrafter** [comment]
Try adding this to your .csproj file:

```xml
<PropertyGroup>
  <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
  <DisableImplicitLibraryPacksFolder>true</DisableImplicitLibraryPacksFolder>
</PropertyGroup>
```
Probably also reload the project and do a `dotnet restore` afterwards.

And please reply back if this helped.

**Reply by kanatos-kun** [reply]
thanks, it's worked !
